questionnaires/tests.py

```diff
@@ -1023,3 +1024,59 @@ def test_csv_export_with_modified_questions(self):
             f'{self.user_submission_02.id},Survey 01,2022-08-31 23:00:00+00:00,question_01,c2\r\n'
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(byte_response.decode(), expected_response)
+
+
+class PollTest(TestCase):
+    def setUp(self):
+        Site.objects.all().delete()
+        self.site = SiteFactory(site_name='IoGT', port=8000, is_default_site=True)
+        self.home_page = self.site.root_page
+        self.poll = PollFactory(parent=self.home_page,)
+
+    def test_drop_down_defaults_to_the_blank_option(self):
+        PollFormFieldFactory(page=self.poll, field_type='dropdown', choices='c1|c2')
+
+        response = self.client.get(self.poll.url)
+        parsed_response = BeautifulSoup(response.content)
+        default_drop_down_option = parsed_response.find('div', {'class': 'quest-item'}).find('select').find('option').text
```

Comment:
I have a feeling that searching HTML markup like this is going to be harder to maintain as the number of tests increases. Beautiful Soup supports [CSS selectors](https://beautiful-soup-4.readthedocs.io/en/latest/#css-selectors), which may be more manageable.

The Selenium tests also use locators, so perhaps there is a way to share these locators across the test suite.